CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+vault-rpm CHANGELOG
```

Comment:
since the changelogs are on the spec files themselves this probably isn't needed